packages/nextjs/components/scaffold-eth/Address.tsx

```diff
@@ -30,6 +31,15 @@ export const Address = ({ address, disableAddressLink, format }: TAddressProps)
     chainId: 1,
     cacheTime: 30_000,
   });
+  const sizeMap = {
+    "3xl": 10,
+    "2xl": 9,
+    xl: 8,
+    lg: 7,
+    base: 5,
+    sm: 4,
+    xs: 3,
+  };
 
```

Comment:
- Let's call this "blockieSizeMap"
- Let's move it outside of the component: (before / after the TAddressProps)